File: docs-examples/rust/nativeexample/examples/connect.rs
use libtaos::*;

fn taos_connect() -> Result<Taos, Error> {
    TaosCfgBuilder::default()
        .ip("localhost")
        .user("root")
        .pass("taosdata")
        // .db("log") // remove comment if you want to connect to database log by default.
        .port(6030u16)
        .build()
        .expect("TaosCfg builder error")
        .connect()
}

fn main() {
    #[allow(unused_variables)]
    let taos = taos_connect().unwrap();
    println!("Connected")
}

File: docs/examples/node/nativeexample/connect.js
const taos = require("td2.0-connector");

var conn = taos.connect({
  host: "localhost",
  port: 6030,
  user: "root",
  password: "taosdata",
});
conn.close();

// run with: node connect.js
// output:
// Successfully connected to TDengine
